docs/diagnostics/MetadataBorders.md

```
@@ -0,0 +1,75 @@
+
+## Примеры
+Получение контактной информации по внутренним правилам должно происходить только через функции
+общего модуля РаботаСКонтактами. Тогда заданные настройки вида
+
+```json
+{"Регистры?Сведений.КонтактнаяИнформация":"CommonModules/РаботаСКонтактами"}
+```
+
+вызовут срабатывание диагностики на запросе с текстом
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)